tests/test_modeling_encoder_decoder.py

```diff
@@ -568,15 +568,27 @@ def test_bert2bert_summarization(self):
 
-        input_ids = tokenizer(ARTICLE, return_tensors="pt").input_ids.to(torch_device)
-        output_ids = model.generate(input_ids)
-        summary = tokenizer.decode(output_ids[0], skip_special_tokens=True)
+        EXPECTED_SUMMARY_SIGMA = """sae was founded in 1856, five years before the civil war. the fraternity has had to work hard to change recently. the university of oklahoma president says the university's affiliation with the fraternity is permanently done. the sae has had a string of members in recent months."""
 
-        self.assertEqual(summary, EXPECTED_SUMMARY)
+        EXPECTED_SUMMARY_AMERICA = """the 2013 america's cup will be faster than ever. the 34th edition of the competition will be held in 2011. the 2013 race will be between multi - hull vessels with wings rather than traditional sails. the new america'' cup will provide increased commercial revenue. the event will also be expanded to a youth america'cup."""
+
+        input_dict = tokenizer(
+            [ARTICLE_SIGMA, ARTICLE_AMERICA],
+            padding="max_length",
+            pad_to_max_length=True,
+            max_length=512,
+            return_tensors="pt",
+        )
+        output_ids = model.generate(
+            input_dict["input_ids"].to(torch_device), attention_mask=input_dict["attention_mask"].to(torch_device)
+        )
+        summary = tokenizer.batch_decode(output_ids, skip_special_tokens=True)
+
+        self.assertEqual(summary, [EXPECTED_SUMMARY_SIGMA, EXPECTED_SUMMARY_AMERICA])
```

Comment:
We need more aggressive slow tests for EncoderDecoderModel now that it's gonna be used in combination with `use_cache`